package.json

```
@@ -22,15 +22,16 @@
     "test": "nyc mocha --async-only"
   },
   "dependencies": {
-    "through2": "^2.0.3"
+    "through2": "^4.0.2"
```

🤔 I need to think about this change:
1. We plan to switch to streamx
2. Bumping 2 major versions is likely breaking somehow for people—usually we'd bump major too

---

```suggestion
    "through2": "^2.0.3"
```
I'm going to undo this change because of #9